src/config/media.config.ts

```diff
@@ -70,6 +75,19 @@ const mediaSchema = Joi.object({
       }),
       otherwise: Joi.optional(),
     }),
+    webdav: Joi.when('use', {
+      is: Joi.valid(BackendType.WEBDAV),
+      then: Joi.object({
+        connectionString: Joi.string().label(
```

Comment:
This and `publicUrl` should both be HTTP(S) URLs, right?
We should probably validate them as such.

Reply:
added